io-sim-classes/src/Control/Monad/Class/MonadTimer.hs

```diff
@@ -42,7 +42,7 @@ fromStart = flip addTime zero
 
 class (Monad m, TimeMeasure (Time m), Show (Time m)) => MonadTime m where
   type Time m :: *
-
+  toInt :: Time m -> m Int
```

Comment:
I've been thinking we should make time durations be a fixed type, not something determined by the monad. This might eliminate the need for this (rather under-specified) operation.

Reply:
@coot :arrow_up: Should we address this now?